builder/builder.go

```diff
@@ -676,7 +676,7 @@ func (b *Builder) getDirBundlesListAsSet(dir string) (bundleSet, error) {
 // writeMixBundleList writes the contents of a bundle set out to the Mix Bundles
 // List file. Values will be in sorted order.
 func (b *Builder) writeMixBundleList(set bundleSet) error {
-	data := []byte(strings.Join(getBundleSetKeysSorted(set), "\n"))
+	data := []byte(strings.Join(getBundleSetKeysSorted(set), "\n") + "\n")
```

Comment:
This says to me "newline delimited list" but the commit message says "command delimited list"

Reply:
Whoops!

Reply:
fixed